test/tests/misc/test_harness_tester/tests

```diff
@@ -1,4 +1,8 @@
 [Tests]
+  [./test_runcommand]
+    type = 'RunCommand'
+    command = 'echo Hello World'
+  [../]
```

Comment:
Funny that you found this file.... I should probably revive this now that we have CIVET!